From: Michael Froman <mjfroman@mac.com>
Date: Tue, 19 Nov 2024 16:16:38 -0600
Subject: (tmp-cherry-pick) Revert "Roll chromium_revision
 ccf648df91..dcb20ee7c2 (1365600:1365760)" (042359fe36)

This reverts commit 1c6d50960e13dc6616bd612385fdbd270755d078.

Reason for revert: speculative revert

Original change's description:
> Roll chromium_revision ccf648df91..dcb20ee7c2 (1365600:1365760)
>
> Change log: https://chromium.googlesource.com/chromium/src/+log/ccf648df91..dcb20ee7c2
> Full diff: https://chromium.googlesource.com/chromium/src/+/ccf648df91..dcb20ee7c2
>
> Changed dependencies
> * src/build: https://chromium.googlesource.com/chromium/src/build/+log/f2790bfa32..ba9b42d40c
> * src/ios: https://chromium.googlesource.com/chromium/src/ios/+log/26dfb6e15c..af24c7d949
> * src/testing: https://chromium.googlesource.com/chromium/src/testing/+log/3ef5641e1f..b86522e058
> * src/third_party: https://chromium.googlesource.com/chromium/src/third_party/+log/c8056c18ac..5c651ea03a
> * src/third_party/depot_tools: https://chromium.googlesource.com/chromium/tools/depot_tools.git/+log/c109945e72..1df84d0a0a
> * src/third_party/perfetto: https://android.googlesource.com/platform/external/perfetto.git/+log/5361e5909e..74fbed7d43
> * src/third_party/r8/d8/cipd: yEomA-IPmb_JtuiEvwgtxRHtSEaICkDY1sDko_rQGO0C..3KCj5eRYCvGGYs5i90pRaeihkzsqgUGc4OkICT8AOlIC
> * src/tools: https://chromium.googlesource.com/chromium/src/tools/+log/8c5814c8d2..63ecf6c58b
> DEPS diff: https://chromium.googlesource.com/chromium/src/+/ccf648df91..dcb20ee7c2/DEPS
>
> No update to Clang.
>
> BUG=None
>
> Change-Id: I758355ae79e03876b8f4dd5aacf10bad0205b972
> Reviewed-on: https://webrtc-review.googlesource.com/c/src/+/364981
> Commit-Queue: Autoroller <chromium-webrtc-autoroll@webrtc-ci.iam.gserviceaccount.com>
> Bot-Commit: Autoroller <chromium-webrtc-autoroll@webrtc-ci.iam.gserviceaccount.com>
> Cr-Commit-Position: refs/heads/main@{#43200}

Bug: None
Change-Id: I3cb43b18de809552dc0f1dfaa1b5521015800cd4
No-Presubmit: true
No-Tree-Checks: true
No-Try: true
Reviewed-on: https://webrtc-review.googlesource.com/c/src/+/365141
Reviewed-by: Mirko Bonadei <mbonadei@webrtc.org>
Commit-Queue: Jeremy Leconte <jleconte@webrtc.org>
Bot-Commit: rubber-stamper@appspot.gserviceaccount.com <rubber-stamper@appspot.gserviceaccount.com>
Cr-Commit-Position: refs/heads/main@{#43207}
---
 DEPS | 18 +++++++++---------
 1 file changed, 9 insertions(+), 9 deletions(-)

diff --git a/DEPS b/DEPS
index 7a95f03edc..f66953e612 100644
--- a/DEPS
+++ b/DEPS
@@ -10,7 +10,7 @@ vars = {
   # chromium waterfalls. More info at: crbug.com/570091.
   'checkout_configuration': 'default',
   'checkout_instrumented_libraries': 'checkout_linux and checkout_configuration == "default"',
-  'chromium_revision': 'dcb20ee7c28712ed658b54b3a2fc3277b6056ac3',
+  'chromium_revision': 'ccf648df915eadad3d27653c32709dfc5b3e5254',
 
   # Fetch the prebuilt binaries for llvm-cov and llvm-profdata. Needed to
   # process the raw profiles produced by instrumented targets (built with
@@ -63,7 +63,7 @@ deps = {
   'src/base':
     'https://chromium.googlesource.com/chromium/src/base@006d78d70b321239c8953bec055e37638e04d900',
   'src/build':
-    'https://chromium.googlesource.com/chromium/src/build@ba9b42d40c98b2d45594931473c52d6485514764',
+    'https://chromium.googlesource.com/chromium/src/build@f2790bfa3238410fbb4bb74b6282f6edb3c1e089',
   'src/buildtools':
     'https://chromium.googlesource.com/chromium/src/buildtools@0a905dcb6d88dbfe4b9b169bf367da9371f72ae3',
   # Gradle 6.6.1. Used for testing Android Studio project generation for WebRTC.
@@ -72,13 +72,13 @@ deps = {
     'condition': 'checkout_android',
   },
   'src/ios': {
-    'url': 'https://chromium.googlesource.com/chromium/src/ios@af24c7d949b3797b1a9dffb7e0be0503ab93a369',
+    'url': 'https://chromium.googlesource.com/chromium/src/ios@26dfb6e15c33512e9ce695d5ed9ea8bcfc2ef8ce',
     'condition': 'checkout_ios',
   },
   'src/testing':
-    'https://chromium.googlesource.com/chromium/src/testing@b86522e0589c5c4ee1787c0dfe5f42441b7c471f',
+    'https://chromium.googlesource.com/chromium/src/testing@3ef5641e1fbbd8fbf2fba9fb0aac929dc3b04d97',
   'src/third_party':
-    'https://chromium.googlesource.com/chromium/src/third_party@5c651ea03af823237a5bc331ca3212d4a189eecc',
+    'https://chromium.googlesource.com/chromium/src/third_party@c8056c18ac5999093ce3c9aa25286d6787e9c6d0',
 
   'src/buildtools/linux64': {
     'packages': [
@@ -251,7 +251,7 @@ deps = {
   'src/third_party/crc32c/src':
     'https://chromium.googlesource.com/external/github.com/google/crc32c.git@d3d60ac6e0f16780bcfcc825385e1d338801a558',
   'src/third_party/depot_tools':
-    'https://chromium.googlesource.com/chromium/tools/depot_tools.git@1df84d0a0a794a76aaef1faae971e1c683d90ca9',
+    'https://chromium.googlesource.com/chromium/tools/depot_tools.git@c109945e729cb3d8c17f306c9cae9d3f98d7d323',
   'src/third_party/ffmpeg':
     'https://chromium.googlesource.com/chromium/third_party/ffmpeg.git@686d6944501a6ee9c849581e3fe343273d4af3f6',
   'src/third_party/flatbuffers/src':
@@ -351,7 +351,7 @@ deps = {
       'condition': 'checkout_android',
   },
   'src/third_party/perfetto':
-    'https://android.googlesource.com/platform/external/perfetto.git@74fbed7d43de71e92e8042f931c76a2662554efc',
+    'https://android.googlesource.com/platform/external/perfetto.git@5361e5909e80f71900ba231872fe0cc008d1e3cb',
   'src/third_party/protobuf-javascript/src':
     Var('chromium_git') + '/external/github.com/protocolbuffers/protobuf-javascript' + '@' + 'e34549db516f8712f678fcd4bc411613b5cc5295',
   'src/third_party/libvpx/source/libvpx':
@@ -399,7 +399,7 @@ deps = {
       'packages': [
           {
               'package': 'chromium/third_party/r8',
-              'version': '3KCj5eRYCvGGYs5i90pRaeihkzsqgUGc4OkICT8AOlIC',
+              'version': 'yEomA-IPmb_JtuiEvwgtxRHtSEaICkDY1sDko_rQGO0C',
           },
       ],
       'condition': 'checkout_android',
@@ -410,7 +410,7 @@ deps = {
     'condition': 'checkout_android',
   },
   'src/tools':
-    'https://chromium.googlesource.com/chromium/src/tools@63ecf6c58b023caa4f453a42b65ece744270af16',
+    'https://chromium.googlesource.com/chromium/src/tools@8c5814c8d24ad90987b4cbd3c69947fc2e461f0e',
 
   'src/third_party/espresso': {
       'packages': [
